### DISCUSSION TITLE
有没有可能组件避免用sap开头

### DISCUSSION THREAD

**Discussion by papwuj** [UI5/webcomponents-react#5550] [Ideas]
有没有可能组件避免用sap开头，这样公司或个人可以没有顾虑的使用；例如react，没有带有facebook等字样，在市场上就能迅速推广开。

**Comment by Lukas742** [comment]
Hi @ruoyaya 

I've translated your text so I and others who can't speak Chinese can understand it as well, I hope the context is not lost in translation:

> Is it possible for components to avoid starting with sap so that companies or individuals can use them without concern; for example, react, which doesn't come with the word facebook, etc., has been able to spread quickly in the marketplace.

Translated using [deepl.com](https://www.deepl.com/translator#zh/en/%E6%9C%89%E6%B2%A1%E6%9C%89%E5%8F%AF%E8%83%BD%E7%BB%84%E4%BB%B6%E9%81%BF%E5%85%8D%E7%94%A8sap%E5%BC%80%E5%A4%B4%EF%BC%8C%E8%BF%99%E6%A0%B7%E5%85%AC%E5%8F%B8%E6%88%96%E4%B8%AA%E4%BA%BA%E5%8F%AF%E4%BB%A5%E6%B2%A1%E6%9C%89%E9%A1%BE%E8%99%91%E7%9A%84%E4%BD%BF%E7%94%A8%EF%BC%9B%E4%BE%8B%E5%A6%82react%EF%BC%8C%E6%B2%A1%E6%9C%89%E5%B8%A6%E6%9C%89facebook%E7%AD%89%E5%AD%97%E6%A0%B7%EF%BC%8C%E5%9C%A8%E5%B8%82%E5%9C%BA%E4%B8%8A%E5%B0%B1%E8%83%BD%E8%BF%85%E9%80%9F%E6%8E%A8%E5%B9%BF%E5%BC%80%E3%80%82).

We don't offer components starting with SAP, but even if we would, it would be fine in my opinion since we're a components library, implementing the [design and UX system](https://experience.sap.com/fiori-design-web/) of SAP so we're opinionated regarding our design and therefore our target audience are people wanting to create an application using the standards of SAP. Also, React and our library are not comparable. React is a library for building user interfaces while we're a components library providing pre-styled components adhering to our UX guidelines, so the two projects have a totally different scope and purpose.

**Comment by papwuj** [comment]
ok thinks
